### PR TITLE
Bugfixes, learned from sys11 setup

### PR DESCRIPTION
**Pull request by lbb**
**What this PR does / why we need it**:
fix

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes:
* Hint's for hostnames
* Cleanup was missing files

**Special notes for your reviewer**:

### CODE REVIEW

File: utils/seed/README.md [hunk 1]
```diff
@@ -27,14 +27,14 @@ CLOUD_PROVIDER_FLAG=openstack
 CLOUD_CONFIG_FILE=./path-to-cloud-conf
 DEFAULT_PRIVATE_IP4_INTERFACE=eth1
 DEFAULT_LOGIN_USER=ubuntu
-ETCD_HOSTNAMES=(etcd-hostname1 etcd-hostname2 etcd-hostname3)
+ETCD_HOSTNAMES=(etcd-hostname1 etcd-hostname2 etcd-hostname3) # Do not use names with dots '.'
```
Comment (alvaroaleman) [inline utils/seed/README.md:30]: Can you elaborate on what is the reason for not using names with dots?

File: utils/seed/cleanup.sh [hunk 2]
```diff
@@ -6,17 +6,21 @@ source ./config.sh
 for ((i = 0; i < ${#MASTER_HOSTNAMES[@]}; i++)); do
         ssh ${DEFAULT_LOGIN_USER}@${MASTER_PUBLIC_IPS[$i]} "sudo kubeadm reset || true"
         ssh ${DEFAULT_LOGIN_USER}@${MASTER_PUBLIC_IPS[$i]} "sudo rm -rf /etc/kubernetes || true"
+        ssh ${DEFAULT_LOGIN_USER}@${MASTER_PUBLIC_IPS[$i]} "rm -rf ~/etc/ || true"
```
Comment (alvaroaleman) [inline utils/seed/cleanup.sh:9]: Why the `|| true`? `rm -rf` will only yield an error if it can not delete a file, e.G. because of permissions but not if there was no file - So if this actually fails I'd assume you want to get notified

File: utils/seed/config.sh [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+ETCD_PUBLIC_IPS=(185.56.130.207 185.56.132.184 185.56.132.185)
 
 POD_SUBNET="10.244.0.0/16" # Canal
-MASTER_LOAD_BALANCER_ADDRS=(192.168.1.89)
+MASTER_LOAD_BALANCER_ADDRS=(185.56.132.187)
 MASTER_HOSTNAMES=(seed-master-1 seed-master-2 seed-master-3)
-MASTER_PRIVATE_IPS=(192.168.1.89 192.168.1.88 192.168.1.9)
-MASTER_PUBLIC_IPS=(192.168.1.89 192.168.1.88 192.168.1.9)
+MASTER_PRIVATE_IPS=(185.56.132.187 185.56.132.189 185.56.132.19)
+MASTER_PUBLIC_IPS=(185.56.132.187 185.56.132.189 185.56.132.19)
```
Comment (alvaroaleman) [inline utils/seed/config.sh:18]: Why did you change all the ip addresses to non-rfc1918 addresses?

File: utils/seed/README.md (hunk 1 above)
Comment (mrIncompetent) [inline utils/seed/README.md:30]: Why?
It's fairly common to use dots in names

**Comment by lbb** [comment]
closes #739

File: utils/seed/README.md (hunk 1 above)
Comment (lbb) [reply utils/seed/README.md:30]: I'll investigate, but currently it leads to erros where just the first word of the fqdn is taken `example.bla.blubb` -> (will put in the SANs) `example`.

Comment (lbb) [reply utils/seed/README.md:30]: Same as above

File: utils/seed/cleanup.sh (hunk 2 above)
Comment (lbb) [reply utils/seed/cleanup.sh:9]: It will throw an error. If the full path does not exists. Which won't happen. I will delete it.

File: utils/seed/config.sh (hunk 3 above)
Comment (lbb) [reply utils/seed/config.sh:18]: This was the latest Sys11 config.

**Comment by lbb** [comment]
PTAL @mrIncompetent @alvaroaleman

File: utils/seed/README.md [hunk 4]
```diff
@@ -64,3 +64,7 @@ kubectl uncordon <node name>
 ```
 
 Repeat for all other nodes one by one.
+
+# Add workers
+
+To add worker nodes simply update to config.sh nodes and execute `install-worker.sh`
```
Comment (mrIncompetent) [inline utils/seed/README.md:70]: This wont work unless install-worker.sh sources config.sh

Comment (lbb) [reply utils/seed/README.md:70]: Fixed sorry for it

File: utils/seed/README.md (hunk 1 above)
Comment (mrIncompetent) [reply utils/seed/README.md:30]: What part is splitting the name by the dot?

Is it kubeadm?
If not, this needs to be fixed

Comment (lbb) [reply utils/seed/README.md:30]: It's kubeadm as fas as we know We don't split anything.

It happens during cert generation.

**Review by mrIncompetent — APPROVED** [review]

**Comment by lbb** [comment]
Closed in favor of 

https://github.com/kubermatic/kubermatic-installer/pull/7
https://github.com/kubermatic/kubermatic/pull/829
